integration_tests/pages/overview/EducationAndTrainingPage.ts

```diff
@@ -33,8 +33,21 @@ export default class EducationAndTrainingPage extends Page {
     return this
   }
 
-  hasFunctionalSkillsDisplayed(): EducationAndTrainingPage {
+  hasFunctionalSkillWithAssessmentScoreDisplayed(
```

Comment:
Improved the naming and functionality of the assertion methods in the page class, so as to give a better API. It means the cypress tests that use them are much more readable 👍